### PR TITLE
Typos, improved grammar and formatting for agreed and implemented RFCs

### PR DESCRIPTION
**Pull request by ligthyear**
While trying to catch up on the RFCs, I noticed some contain typos, were hard to read (with an editor) or had some formatting differences (like unneeded wrapping or missing markup). This PR contains these fixes I made to the agreed and implemented RFCs. None of the fixes changes the actual content or meaning of any of RFCs.

---
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/rfcs/119)

### CODE REVIEW

**Comment by ligthyear** [comment]
Just let me know if that is something you'd like to see or if there were any assumptions I made (like no-wrap), which you'd want to have a different agreement upon. If you like what I did here, I am glad to continue doing this for the remaining RFCs, too. Just let me know :) .

**Comment by dirvine** [comment]
Absolutely, this is extremely helpful.

File: agreed/0004-Farm-attempt/0004-Farm-attempt.md [hunk 1]
```diff
@@ -32,22 +32,19 @@ ON receipt of a `Get` request for `ImmutableData` the `DataManager` calculates t
 1. PmidNodes -> Registers an Optional (if set by user) wallet address on any store of data to it
 2. App Developer -> App developers will include wallet address in any `Get` request
 3. Publisher -> As data is stored a wallet address of the owner is stored if this is first time seen on the network (stored in DM account for the data element)
-4. Core development -> Initially every node will be aware of a hard coded wallet address for core development. This will likely lead to a multisig wallet.
+4. Core development -> Initially every node will be aware of a hard coded wallet address for core development. This will likely lead to a multi-sign wallet.
```
Comment (rossmuir) [inline agreed/0004-Farm-attempt/0004-Farm-attempt.md]: I believe "multisig" is correct here

**Comment by rossmuir** [comment]
Reviewed 9 of 9 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/rfcs/119)*

**Comment by rossmuir** [comment]
@ligthyear Big thank you :+1:  as @dirvine said this is extremely helpful.

**Comment by ligthyear** [comment]
*[agreed/0004-Farm-attempt/0004-Farm-attempt.md, line 35 \[r1\]](https://reviewable.io:443/reviews/maidsafe/rfcs/119#-KGOMvQR9PST5OCPCpCJ-r1-35:-KGlpgQkmJic-eLBTjfZ:1320125615) ([raw file](https://github.com/maidsafe/rfcs/blob/2016510771876dcb9c8bb3ae70fce63660f13a43/agreed/0004-Farm-attempt/0004-Farm-attempt.md#L35)):*
I was hesitating on that actually. But missing any definition I wasn't sure if what "multisig" is (does it stand for multiple signature? or multiple signing?) and went for the smalles amount of typos for it to make sense. If it is meant to be about "multi signature", then I am happy to go for that, too (actually spelling it out would be better, lacking a good glossary or word definition list though).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/rfcs/119)*

File: agreed/0004-Farm-attempt/0004-Farm-attempt.md (hunk 1 above)
Comment (frabrunelle) [inline agreed/0004-Farm-attempt/0004-Farm-attempt.md]: Yes, "multisig wallet" is definitely more common than "multi-sign". The other option is to say "multisignature wallet".

https://en.wikipedia.org/wiki/Multisignature

**Pull request merged**